GameDev/PlayerCollidableBehaviour.cpp

```diff
@@ -0,0 +1,28 @@
+#include "PlayerCollidableBehaviour.h"
+
+
+PlayerCollidableBehaviour::PlayerCollidableBehaviour()
+{
+}
+
+
+PlayerCollidableBehaviour::~PlayerCollidableBehaviour()
+{
+}
+void PlayerCollidableBehaviour::Hit(CollidableBehaviour* cb){
+	if (cb){
+		switch (cb->GetActor()->GetType()){
```

Comment:
is er niet een risico dat deze switch case in de toekomst heel groot gaat worden?